city_scrapers/spiders/chi_ssa_23.py

```diff
@@ -0,0 +1,143 @@
+    start_urls = ["https://www.lincolnparkchamber.com/clark-street-ssa-administration/"]
+    location = {
+        "name": "Lincoln Park Chamber of Commerce",
+        "address": "2468 N. Lincoln Chicago, IL 60614"
+    }
+    # Each meeting takes place on Wednesday 4 PM
+    meeting_day = 'Wednesday'
+    time = '4 pm'
+
+    def parse(self, response):
+        # Due to the current lack of documents for the meetings of 2020
+        # no assumption is made regarding the expected HTML page format
+        # once these are uploaded.
+        h4s = response.xpath('//h4')
+        meeting_years = list()
+
+        for entry in h4s.xpath('./text()').getall():
+            # Do not allow duplicates
+            if entry[0:4] not in meeting_years:
+                meeting_years.append(entry[0:4])
+
+        # General meeting description is mentioned just after the H4 for the current year
+        general_desc = h4s.xpath('following-sibling::p[1]//em//text()').extract_first()
+
+        # List containing all meeting dictionaries
+        meetings = list()
+
+        link_collection = []
+        for h4 in h4s[1:]:
+            # Get the links for each meeting
+            link_collection.append(h4.xpath('following-sibling::p[1]//a'))
+
+        last_year = datetime.today().replace(year=datetime.today().year - 1)
+
+        for year_counter, year in enumerate(meeting_years):
+
+            if year_counter == 0:
+                for item in h4s.xpath('following-sibling::ol[1]//li//text()').extract():
+                    start, end = self._parse_start_end(item, meeting_years[0])
+
+                    meetings.append({
+                        'start': start,
+                        'end': end,
+                        # Currently no meetings from 2020 have links
+                        'links': None
+                    })
+
+            # Always multiply the counter with two as each year has usually two links
+            else:
+                logging.log(logging.DEBUG, link_collection[year_counter])
+                for item_counter, item in enumerate(link_collection[2 * (year_counter - 1)]):
+                    # Get the text part
+                    item_str = item.xpath('./text()').extract_first()
+                    logging.log(logging.DEBUG, item_str)
+                    start, end = self._parse_start_end(item_str, year)
+
+                    links = [item.xpath('@href').extract_first()]
+                    # Try to see if we have a second link
+                    try:
+                        links.append(
+                            link_collection[2 * (year_counter - 1) +
+                                            1][item_counter].xpath('@href').extract_first()
+                        )
+                    except IndexError:
+                        # No problem, this means that this is an older entry and
+                        # there is no second link to this element
+                        links.append(None)
```

Comment:
We don't want to add `None` to any list of links. If a value is invalid we can leave it out

Reply:
Right, I changed that part to pass.